pkgs/development/compilers/circt/default.nix

```diff
@@ -67,6 +67,11 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "lib" "dev" ];
 
   postInstall = ''
+    ln -s ${circt-llvm.dev}/include/* $out/include/
+
+    find ${circt-llvm.lib}/lib -type f -exec ln -s '{}' $out/lib/ \;
```

Comment:
Should we use `-maxdepth 1` to avoid involving .o stuff?

Besides, only files (not symlinks) are linked, will it cause problems?

Reply:
Yes it might cause error....I misuse the find util.